src/kibana/components/vislib/visualizations/_point_series_chart.js

```diff
@@ -15,6 +15,30 @@ define(function (require) {
       PointSeriesChart.Super.apply(this, arguments);
     }
 
+    PointSeriesChart.prototype._stackMixedValues = function (stackCount) {
+      var currentStackOffsets = [0, 0];
+      var currentStackIndex = 0;
+
+      return function (d, y0, y) {
+        var firstStack = currentStackIndex % stackCount === 0;
+        var lastStack = ++currentStackIndex === stackCount;
+
+        if (firstStack) {
+          currentStackOffsets = [0, 0];
+        }
+
+        if (lastStack) currentStackIndex = 0;
```

Comment:
The reason I added the counter reset was because I suspected that `stackCount` could change, while `currentStackIndex` would just keep incrementing infinitely. That means that the modulo logic would fail, since the index and the stack count can drift - resetting the index at the end of the stack would help avoid that (since it should always be 0 when starting out). Doing so should also make is safe to remove the modulo and just check for `index === 0`